services/worker-manager/src/provisioner.js

```diff
@@ -64,8 +65,10 @@ class Provisioner {
    */
   async provision() {
     if (this.provisioningLoopAlive) {
-      this.monitor.notice('loop-interference', {});
-      return;
+      this.monitor.warning('loop-interference', {});
+      // should be treated as terminal error
+      // to let the pod to restart and avoid getting stuck in a loop
+      throw new ApiError('provision loop interference');
```

Comment:
I haven't looked yet, but I wonder if there are string constants for errors that we could use here (or you could make a new one and use it here if that _is_ the practice in the repo). If constants for error messages aren't used, I'd call this good! Just wanna be consistent.

Reply:
I like the idea, but I haven't seen much constants either. In other places it was also like this. 
It is different for monitor though, where you need to register proper format.